nifi-nar-bundles/nifi-standard-services/nifi-oauth2-provider-bundle/nifi-oauth2-provider-service/src/main/java/org/apache/nifi/oauth2/StandardOauth2AccessTokenProvider.java

```diff
@@ -69,6 +71,27 @@ public class StandardOauth2AccessTokenProvider extends AbstractControllerService
         .expressionLanguageSupported(ExpressionLanguageScope.VARIABLE_REGISTRY)
         .build();
 
+    public static AllowableValue BASIC_AUTHENTICATION = new AllowableValue(
+        "basic",
+        "Basic Authentication",
+        "Send client authentication as HTTP Basic authentication"
+    );
+
+    public static AllowableValue REQUEST_BODY = new AllowableValue(
+        "request_body",
+        "Request Body",
+        "Send client authentication in request body. Note: users should prefer Basic Authentication unless authorization server does not support it."
+    );
+
+    public static final PropertyDescriptor CLIENT_AUTHENTICATION_STRATEGY = new PropertyDescriptor.Builder()
+        .name("client-authentication-strategy")
+        .displayName("Client Authentication Strategy")
+        .description("Authentication strategy to use for client authentication.")
```

Comment:
A little awkward in the wording. Should be something like `Strategy for authenticating the client against the OAuth2 token provider service`.

Reply:
I agree with the suggested improvement, it is clearer/

Reply:
Updated with suggested wording.